riscv/triggers.cc

```diff
@@ -37,6 +38,7 @@ bool mcontrol_t::tdata1_write(processor_t * const proc, const reg_t val) noexcep
   }
   auto xlen = proc->get_xlen();
   dmode = get_field(val, MCONTROL_DMODE(xlen));
+  hit = get_field(val, CSR_MCONTROL_HIT);
```

Comment:
Why is this `#define` of a different form than all the other fields?

Reply:
This one comes from debug_defines.h, which is more correct and up-to-date than encoding.h. I'm planning to move all macros to the debug_defines.h ones, but before that I want https://github.com/riscv/riscv-debug-spec/pull/724 to merge over there.